Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3003,7 +3003,7 @@ RefPtr<StyleValue> Parser::parse_counter_value(TokenStream<ComponentValue>& toke
         auto& function = token.function();
         TokenStream function_tokens { function.values() };
         auto function_values = parse_a_comma_separated_list_of_component_values(function_tokens);
-        if (function_values.is_empty() || function_values.size() > 3)
+        if (function_values.is_empty() || function_values.size() < 2 || function_values.size() > 3)
```

Comment:
Minor thing, but this makes the `is_empty` check redundant
```c++
if (function_values.size() < 2 || function_values.size() > 3)
```

Reply:
Good point! Fixed.